components/Support/Support.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Page } from '@/components/Page/Page'
+
+export default function Support() {
+  return (
+    <Page>
+      <section className="text-center">
+        <h1>Get Involved</h1>
+        <h2>
+          Not interested in hacking but still want to participate? Great! We
+          would love to have you on board. Fill out the forms below if you'd
+          like to help hackers throughout the day.
+        </h2>
+        <p className="text-left">
+          <p className="font-medium">Mentors</p>
+          <p>
+            Mentors are in charge of helping the hackers with new technologies
+            and working through any bugs or obstacles they encounter. We
+            recommend this position if you are well versed in a particular stack
+            to offer the best guidance.
+          </p>
```

Comment:
can u make each of these little sections ("mentors", "volunteers", etc.) a component, and pass in the details in as a parameter